lib/order.rb

```diff
@@ -0,0 +1,21 @@
+class Order
+
+  attr_reader :order
+
+  def initialize
+    @order = {}
+  end
+
+  def add item, number, menu
+    line= menu.select_food(item.to_s)
+    line.each{|k,v| v[1]=number}
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.